kolibri/core/tasks/api_urls.py

```diff
@@ -1,10 +1,12 @@
 from rest_framework import routers
 
+from .api import BaseViewSet
 from .api import FacilityTasksViewSet
 from .api import TasksViewSet
 
 router = routers.SimpleRouter()
 router.register("tasks", TasksViewSet, base_name="task")
 router.register("facilitytasks", FacilityTasksViewSet, base_name="facilitytask")
+router.register("", BaseViewSet, base_name="managetask")
```

Comment:
I would prefer not to use a blank base URL path here, it feels like an anti-pattern to me.

We can have this under `managetask` for now, but as we finish migrations of tasks to the new API, I would expect that `BaseViewSet` will be renamed `TasksViewSet` and that this will all be under `"tasks"` with a basename of `"tasks"`, as at that point, we can dispense with the other two endpoints!

Reply:
@rtibbles the only reason to use a blank base URL path was to have urlconf generated as `/api/tasks/` not `/api/tasks/tasks/`. Don't you think the former looks more beautiful?

Reply:
The general pattern we have for API URLs has <appname>/<endpoint> - so I'd rather keep with that, even if it looks repetitive in this case.

Reply:
Okay - so I'll remove the new endpoint that I've added. Since, `TasksViewSet` inherits from `BaseViewSet`, until `TasksViewSet` is alive it'll handle the endpoints. After migration, I'll remove the two viewsets that we have and will rename `BaseViewSet` to `TasksViewSet`.